spec/features/signingin_spec.rb

```diff
@@ -0,0 +1,25 @@
+feature "Signing in:" do
+  context "When signing up:" do
+
+    scenario "signing up should increase the user count by 1" do
+      expect {test_sign_up}.to change(User, :count).by(1)
+    end
+
+    scenario "signing up should take the user to their Peeps page" do
+      test_sign_up
+      expect(current_path).to eq('/peeps')
+    end
+
+    scenario "a user should not be created if there is the passwords do not match" do
+      expect{
```

Comment:
Style/BlockDelimiters: Avoid using `{...}` for multi-line blocks. (https://github.com/bbatsov/ruby-style-guide#single-line-blocks)